server/rest/src/test/java/org/infinispan/rest/resources/CacheV2ResourceTest.java

```diff
@@ -643,6 +644,11 @@ public void testSearchStatistics() {
 
       // Clear all stats
       RestResponse response = join(cacheClient.clearSearchStats());
```

Comment:
shouldn't this be invoked only when `security==false`?

Reply:
All of the tests rely on zeroed stats so it needs to be called.

Reply:
true, but you are invoked `clearSearchStats()` twice when `security==true`; here and other as "admin"